components/availability/src/Availability.svelte

```diff
@@ -548,6 +595,31 @@
     return classes.join(" ");
   }
   //#endregion Attendee Overlay
+
+  // #region Date Change
+  function goToNextDate() {
+    if (show_as_week) {
+      start_date = timeWeek.offset(endDay, 1);
```

Comment:
IIRC there's a limitation with any `internalProps` (including `start_date`), where they shouldn't have their value set by the component internally - it causes some sort of bug if the user tries to update that prop externally, I think reactivity might break. Can you use an internal var instead of modifying `start_date` directly?

Reply:
After discussion w/ Oliver, I'm going to make this an item to resolve later. The current PR will disallow prev/next functionality in the event that, post-render, `start_date` is affected externally and passed down.